specification/attributes/column_naming_convention.md

```diff
@@ -27,7 +27,33 @@ Naming convention for columns appearing in billing data.
 - Custom columns SHOULD follow the same rules as FOCUS columns listed above.
 - All custom column IDs MUST start with a vendor prefix string and an underscore (e.g., `abc_ColumnName`).
   - Vendor prefixes MUST be lowercase, alphanumeric strings and SHOULD NOT be longer than 10 characters in length.
-  - Vendor prefixes MUST be representative of the company or brand that generated the column.
+  - Vendor prefixes MUST much match an entry in the Vendor Prefix List as defined in this appendix when an vendor organization is present therein.
+  - Columns making use of a vendor prefix MUST be generated and managed by the Management Organization in the Vendor Prefix List.
+  - Providers and managing organzition names SHOULD be listed in the Vendor Prefix List as defined in this appendix.
+  - Managing organizations SHOULD public a specification document for custom columns including the vendor prefix, column name, and the version of the FOCUS specification each column is compatible with.
+
+### Vendor Prefix List
+
+The following table lists the vendor prefixes that are currently registered in the Vendor Prefix List as part of the FOCUS specification.
+
+| Vendor Prefix | Provider Name         | Managing Organziation |
```

Comment:
If we're going this far, I would love to add a column for documentation along with a requirement that providers MUST (?) document their added columns. That would provide some real value for practitioners. (Until the link gets broken 😓)